erpnext/erpnext_integrations/doctype/affirm_settings/affirm_settings.json

```diff
@@ -0,0 +1,131 @@
+{
+ "actions": [],
+ "creation": "2018-02-22 15:33:14.335573",
+ "doctype": "DocType",
+ "document_type": "Setup",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "is_sandbox",
+  "sb_settings",
+  "public_api_key",
+  "private_api_key",
+  "cb_header",
+  "live_checkout_url",
+  "live_api_url",
+  "sb_sandbox_setting",
+  "public_sandbox_api_key",
+  "private_sandbox_api_key",
+  "cb_sandbox",
+  "sandbox_checkout_url",
+  "sandbox_api_url",
+  "shipping_sb"
+ ],
+ "fields": [
+  {
+   "fieldname": "public_api_key",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Public API Key",
+   "reqd": 1
+  },
+  {
+   "fieldname": "private_api_key",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Private API Key",
+   "reqd": 1
+  },
+  {
+   "fieldname": "cb_header",
+   "fieldtype": "Column Break"
+  },
+  {
+   "default": "1",
+   "fieldname": "is_sandbox",
+   "fieldtype": "Check",
+   "label": "Sandbox Mode"
+  },
+  {
+   "fieldname": "sb_settings",
+   "fieldtype": "Section Break",
+   "label": "URLS"
+  },
+  {
+   "default": "https://cdn1.affirm.com/js/v2/affirm.js",
+   "fieldname": "live_checkout_url",
+   "fieldtype": "Data",
+   "label": "Live Checkout Url",
+   "reqd": 1
+  },
+  {
+   "default": "https://api.affirm.com/api/v2",
+   "fieldname": "live_api_url",
+   "fieldtype": "Data",
+   "label": "Live API Url",
+   "reqd": 1
+  },
+  {
+   "default": "https://cdn1-sandbox.affirm.com/js/v2/affirm.js",
+   "fieldname": "sandbox_checkout_url",
+   "fieldtype": "Data",
+   "label": "Sandbox Checkout Url",
+   "reqd": 1
+  },
+  {
+   "default": "https://sandbox.affirm.com/api/v2",
+   "fieldname": "sandbox_api_url",
+   "fieldtype": "Data",
```

Comment:
Rather than taking input from the user for the api url, I think you can hard code this into code. When sandbox mode is selected you can fetch sandbox url